app/src/main/res/values/strings.xml

```diff
@@ -1795,7 +1796,7 @@
     <string name="voice_search_explainer">Speak now</string>
     <!--  The error message in edit login view when a duplicate username exists. -->
     <string name="saved_login_duplicate">A login with that username already exists</string>
-    <!-- This is the hint text that is shown inline on the hostname field of the create new login page. -->
+    <!-- This is the hint text that is shown inline on the hostname field of the create new login page. 'https://www.example.com' intentionally hardcoded here -->
```

Comment:
@Delphine thanks for the review, I added that `example.com` is intentionally hardcoded